sklearnex/dispatcher.py

```diff
@@ -201,6 +237,54 @@ def get_patch_map_core(preview=False):
         ]
         mapping["logisticregression"] = mapping["log_reg"]
 
+        # Ridge
+        mapping.pop("ridge")
+        mapping["ridge"] = [
+            [
+                (
+                    linear_model_module,
+                    "Ridge",
+                    Ridge_sklearnex,
+                ),
+                None,
+            ]
+        ]
+
+        # manifold
+        mapping.pop("tsne")
+        mapping["tsne"] = [
+            [
+                (manifold_module, "TSNE", TSNE_sklearnex),
+                None,
+            ]
+        ]
+
+        # metrics
+        mapping.pop("distances")
+        mapping.pop("roc_auc_score")
+        mapping["distances"] = [
+            [
+                (metrics_module, "pairwise_distances", pairwise_distances_sklearnex),
+                None,
+            ]
+        ]
+        mapping["pairwise_distances"] = mapping["distances"]
```

Comment:
Why is "distances" key needed?

Reply:
I am not sure why is it needed. But it existed in daal4py mapping.
For this kind of cases I just made consistent changes: see https://github.com/intel/scikit-learn-intelex/blob/324c9ccf928d439c64a2759f60c5bcc782dd5a81/sklearnex/dispatcher.py#L238C9-L238C59